charts/prow/templates/crier_deployment.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "crier" . }}
+  labels:
+      chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+      app: {{ template "crier" . }}
+spec:
+  replicas: {{ .Values.crier.replicaCount }}  # This will race with itself. Only run one at a time.
```

Comment:
Should this just be hardcoded to 1, since noone should ever change it? Same for the others with this comment.